<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>history</class>
 <widget class="QWidget" name="history">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="back_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="down_button_label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>430</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="up_button_label">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>430</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="history_title_label">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>20</y>
     <width>241</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>History</string>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="usb_button_label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>200</y>
     <width>621</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>621</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>300</y>
     <width>621</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>350</y>
     <width>621</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>400</y>
     <width>621</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="image_screen">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1031</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="text_1_label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>591</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>1CH-2CM, 60W(- Watt), 6Min(- Time), 870Kcal(- Energy)</string>
   </property>
  </widget>
  <widget class="QLabel" name="text_2_label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>591</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>1CH-2CM, 60W(- Watt), 6Min(- Time), 870Kcal(- Energy)</string>
   </property>
  </widget>
  <widget class="QLabel" name="text_3_label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>260</y>
     <width>591</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>1CH-2CM, 60W(- Watt), 6Min(- Time), 870Kcal(- Energy)</string>
   </property>
  </widget>
  <zorder>image_screen</zorder>
  <zorder>back_label</zorder>
  <zorder>down_button_label</zorder>
  <zorder>up_button_label</zorder>
  <zorder>history_title_label</zorder>
  <zorder>usb_button_label</zorder>
  <zorder>line</zorder>
  <zorder>line_2</zorder>
  <zorder>line_3</zorder>
  <zorder>line_4</zorder>
  <zorder>line_5</zorder>
  <zorder>text_1_label</zorder>
  <zorder>text_2_label</zorder>
  <zorder>text_3_label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
